src/domain/usecases/NotifyWatchStaffUseCase.ts

```diff
@@ -0,0 +1,37 @@
+import { Ref } from "../entities/Ref";
+import {
+    NotificationOptions,
+    NotificationRepository,
+} from "../repositories/NotificationRepository";
+import { FutureData } from "../../data/api-futures";
+
+export class NotifyWatchStaffUseCase {
+    constructor(private notificationRepository: NotificationRepository) {}
+
+    public execute(
+        outbreakKey: string,
+        notificationData: NotificationOptions,
+        userGroups: Ref[]
+    ): FutureData<void> {
+        return this.notificationRepository.save({
+            subject: `New Outbreak Alert: ${outbreakKey} in zm Zambia Ministry of Health`,
```

Comment:
get this notification text reviewed by PM.

Reply:
@bhavananarayanan has confirmed that this is fine